<launch>  

  <arg name="dataRoot" value="$(env HOME)" />
  <arg name="tabletop_segmentation_service" value="tabletop_segmentation" />

  <node name="pr2_pbd_interaction" pkg="pr2_pbd_interaction" type="interaction.py" output="screen">
    <param name="dataRoot" value="$(arg dataRoot)" />
    <param name="actionsRoot" value="$(arg dataRoot)/pbd_actions" />
    <param name="isReload" value="false" />
    <param name="experimentNumber" value="0" />
    <param name="tabletop_segmentation_service" value="$(arg tabletop_segmentation_service)" />
  </node>
</launch>
